api/api-doc/hinted_handoff.json

```diff
@@ -7,6 +7,61 @@
       "application/json"
    ],
    "apis":[
+      {
+         "path":"/hinted_handoff/sync_point",
+         "operations":[
+            {
+               "method":"POST",
+               "summary":"Creates a hints sync point. It can be used to wait until hints between given nodes are replayed. A sync point allows you to wait for hints accumulated at the moment of its creation - it won't wait for hints generated later.",
```

Comment:
Please add an explanation here about how the sync point gets garbage collected later.

(real reason is that I'm too lazy to go back and hunt for it myself)

Reply:
Done (mentioned that sync point is described entirely by its ID and no state is kept on the Scylla side)

Reply:
Wow, that is nice. Maybe I knew it before but I managed to forget it.

Reply:
Some context: https://github.com/scylladb/scylla/pull/8728#issuecomment-865026723

Reply:
Ah, read that comment but then missed everything that followed, so it did not stick in my mind.